project/application/src/components/TeamSwitcher/index.js

```diff
@@ -45,7 +46,21 @@ class TeamSwitcher extends Component {
             }),
         }).then(res =>
             res.json().then(response => {
-                window.location.reload(true);
+                fetch(AppConfig.apiURL + `team/current/?userId=${getUserIdFromLocalStorage()}`).then(res =>
+                    res.json().then(response => {
+                        this.setState({
+                            currentTeamId: response.data.user_team[0].team.id,
```

Comment:
I see that now we send a request to the server to change current team & update info on the front. So we need to remove lines from 27 to 36